test/tlv/TlvSchemaTest.ts

```diff
@@ -25,6 +25,7 @@ import {
     TlvInt8, TlvInt16, TlvInt32, TlvInt64,
     TlvFloat, TlvDouble 
 } from "../../src/tlv/TlvNumber.js";
+import { TlvWrapper } from "../../src/matter.js";
```

Comment:
This import via `matter.js` doesn't match the style of other imports in this file.